registry/storage/driver/gcs/gcs.go

```diff
@@ -160,6 +165,11 @@ func FromParameters(parameters map[string]interface{}) (storagedriver.StorageDri
 			return nil, err
 		}
 		ts = jwtConf.TokenSource(context.Background())
+
+		gcs, err = storage.NewClient(context.Background(), option.WithCredentialsFile(fmt.Sprint(keyfile)))
```

Comment:
generally, do we conside to enable retry of the client?
https://pkg.go.dev/cloud.google.com/go/storage#BucketHandle.Retryer